### PR TITLE
fix(core-tester-cli): fee parsing & display output in ark

### PR DESCRIPTION
**Pull request by dated**
## Proposed changes

The recent changes to `core-tester-cli` caused the random fee calculation to not work as expected when a fee-range was provided, always returning `0` or `1`. This also changes the log outputs to display the numeric values in ARK instead of ARKtoshis.

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)

## Checklist

- [x] I have read the [CONTRIBUTING](https://docs.ark.io/developers/guidelines/contributing.html) documentation
- [x] Lint and unit tests pass locally with my changes

### CODE REVIEW

File: packages/core-tester-cli/lib/commands/command.js [hunk 1]
```diff
@@ -287,6 +287,15 @@ module.exports = class Command {
     return bignumify(ark * Math.pow(10, 8))
   }
 
+  /**
+   * Convert Arktoshi to ARK.
+   * @param  {Bignum} arktoshi
+   * @return {String}
+   */
+  static __arktoshiToArk (arktoshi) {
+    return (arktoshi.toNumber() / Math.pow(10, 8)).toFixed()
```
Comment (faustbrian) [inline packages/core-tester-cli/lib/commands/command.js]:
```suggestion
    return arktoshi.div(1e8).toFixed()
```

**Comment by supaiku0** [comment]
Printing Arktoshis in Ark can be done with: https://github.com/ArkEcosystem/core/blob/develop/packages/crypto/lib/utils/format-arktoshi.js

File: packages/core-tester-cli/lib/commands/command.js [hunk 2]
```diff
@@ -179,15 +179,15 @@ module.exports = class Command {
...
+      const feeRange = fee.split('-').map(f => +bignumify(Number(f) * Math.pow(10, 8)).toFixed())
       if (feeRange[1] < feeRange[0]) {
         return feeRange[0]
       }
 
       return bignumify(Math.floor((Math.random() * (feeRange[1] - feeRange[0] + 1)) + feeRange[0]))
     }
 
-    return bignumify(fee)
+    return bignumify(fee * Math.pow(10, 8))
```
Comment (faustbrian) [inline packages/core-tester-cli/lib/commands/command.js]: https://github.com/ArkEcosystem/core/pull/1284

File: packages/core-tester-cli/lib/commands/command.js [hunk 3]
```diff
@@ -179,15 +179,15 @@ module.exports = class Command {
    */
   static parseFee (fee) {
     if (typeof fee === 'string' && fee.indexOf('-') !== -1) {
-      const feeRange = fee.split('-').map(f => +bignumify(f).toFixed())
+      const feeRange = fee.split('-').map(f => +bignumify(Number(f) * Math.pow(10, 8)).toFixed())
```
Comment (faustbrian) [inline packages/core-tester-cli/lib/commands/command.js]: https://github.com/ArkEcosystem/core/pull/1284

Comment (faustbrian) [reply packages/core-tester-cli/lib/commands/command.js]: `+bignumify(Number(f)).times(1e8).toFixed()`

File: packages/core-tester-cli/lib/commands/command.js (hunk 2 above)
Comment (faustbrian) [reply packages/core-tester-cli/lib/commands/command.js]: `bignumify(fee).times(1e8)`

**Comment by dated** [comment]
Thanks to both of you!

**Pull request merged**